File: SiteVisitsNetCore2020Web/ClientApp/src/app/models/Visitor.ts
export class Visitor {
  //id: number;
  id: string;
  label: string;
  note: string;

  constructor(options: {
    //id: number,
    id: string,
    label: string,
    note: string
  }) {
    if (options) {
      this.id = options.id;
      this.label = options.label;
      this.note = options.note;
    }
  }
}

File: SiteVisitsNetCore2020Web/Models/Visitor.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace SiteVisitsNetCore2020Web.Models
{
    public class Visitor
    {
        //public int Id { get; set; }
        public Guid Id { get; set; }
        public string Label { get; set; }
        public string Note { get; set; }
        public DateTime FirstOccurrence { get; set; }
        public DateTime DateCreated { get; set; }
    }
}
